v4/Http/Controllers/SurveyController.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace v4\Http\Controllers;
+use Ramsey\Uuid\Uuid;
+use Ushahidi\App\Validator\LegacyValidator;
+use v4\Models\Attribute;
+use v4\Models\Survey;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Validator;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\Rule;
+
+
+class SurveyController extends V4Controller
+{
+    protected static function getRules() {
+        return [
+            'name' => [
+                'required',
+                'min:2',
+                'max:255',
+                'regex:' . LegacyValidator::REGEX_STANDARD_TEXT
+            ],
+            'description' => [
+                'string',
+                'nullable'
+            ],
+            //@TODO find out where this color validator is implemented
+            //[['color']],
```

Comment:
note to self do this with  getColorAttribute much like I did can_create with getCanCreateAttribute :D laravel yay